test/js/dimension-handler.js

```diff
@@ -71,4 +71,8 @@ suite('dimension-handler', function() {
     assert.isUndefined(webAnimations1.consumeLengthOrPercent('(10px)'));
     assert.isUndefined(webAnimations1.consumeLengthOrPercent('calc(10px,10px)'));
   });
+  test('interpolation of more specific properties', function() {
+    assert.equal(webAnimations1.propertyInterpolation('strokeDashoffset', '10px', '50px')(0.25), '20px');
```

Comment:
This should be able to interpolate multi valued values e.g. '10px 20px' to '50px 60px'.
If it doesn't can you file an issue to add support for it? I won't block this patch on supporting that right away.

lgtm.